package-lock.json

```diff
@@ -1246,6 +1246,21 @@
       "integrity": "sha1-WWZ/QfrdTyDMvCu5a41Pf3jsA2c=",
       "dev": true
     },
+    "async": {
```

Comment:
Do you know why package-lock.json has changed but package.json hasn't?

Reply:
We've looked into this and we don't understand why it's changed. :cry: But everything seems to work, so let's just accept it and keep an eye out for it happening again in a branch with fewer cross-merges.